force-app/main/default/objects/Product2/Product2.object-meta.xml

```diff
@@ -84,6 +84,20 @@
         <formFactor>Small</formFactor>
         <type>Default</type>
     </actionOverrides>
+    <actionOverrides>
+        <actionName>PreviewProductAction</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>PreviewProductAction</actionName>
+        <formFactor>Large</formFactor>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>PreviewProductAction</actionName>
+        <formFactor>Small</formFactor>
+        <type>Default</type>
+    </actionOverrides>
```

Comment:
I guess this change was made due to changes in metadata logic by Salesforce and we can push it. Only wanted to comment it to find this again, if we encounter issues in merging.